Sources/MapboxDirections/DirectionsResult.swift

```diff
@@ -139,6 +143,11 @@ open class DirectionsResult: Codable {
      */
     open var expectedTravelTime: TimeInterval
     
+    /**
+     The route’s typical travel time, measured in seconds.
```

Comment:
Should we mention any caveats to this property, such as the ones in the `expectedTravelTime` documentation above?

* Does it work in every profile or just driving profiles?
* Presumably these times are still subject to the vagaries of ferry schedules.
* The developer shouldn’t assume the user would travel along the route at a fixed speed. Does the Directions API expose an attribute option (attribute) for typical travel times as it does for expected travel times?

Reply:
I've added more details about `duration_typical`:
- `duration_typical` is only available when using `driving-traffic` profile.
- Tested with route which contains ferry and both `duration` and `duration_typical` are affected, mentioned this in docs.
- I'll mention that user shouldn't make an assumption that travel along the route will be at a fixed speed. I also don't see that `duration_typical` is supported by `AttributeOptions`.

@danpaz, not sure if you're the right person to ask, but would it be possible to update documentation in https://docs.mapbox.com/api/navigation/#retrieve-directions to contain `duration_typical` related info?